packages/x-charts/src/hooks/useSvgRef.ts

```diff
@@ -1,3 +1,4 @@
+'use client';
```

Comment:
I'm not sure here: https://github.com/mui/material-ui/pull/41956#discussion_r1752127870.